frontend/src/metabase/css/core/colors.module.css

```diff
@@ -54,38 +56,41 @@
     var(--mb-color-bg-white) 15%,
     transparent
   );
-  --mb-color-success: #84bb4c;
-  --mb-color-summarize: #88bf4d;
-  --mb-color-error: #ed6e6e;
-  --mb-color-danger: #ed6e6e;
-  --mb-color-text-dark: #4c5773;
-  --mb-color-text-medium: #696e7b;
-  --mb-color-text-light: #949aab;
-  --mb-color-bg-black: #2e353b;
-  --mb-color-bg-dark: #93a1ab;
-  --mb-color-bg-medium: #edf2f5;
-  --mb-color-bg-light: var(--mb-base-color-gray-5);
+  --mb-color-success: var(--mb-base-color-palm-50);
+  --mb-color-summarize: var(--mb-base-color-palm-50);
+  --mb-color-error: var(--mb-base-color-lobster-60);
+  --mb-color-danger: var(--mb-base-color-lobster-60);
+  --mb-color-text-dark: var(--mb-base-color-orion-80);
+  --mb-color-text-medium: var(--mb-base-color-orion-alpha-60);
+  --mb-color-text-light: var(--mb-base-color-orion-alpha-40);
+  --mb-color-bg-black: var(--mb-base-color-orion-100);
+  --mb-color-bg-dark: var(--mb-base-color-orion-alpha-80);
+  --mb-color-bg-medium: var(--mb-base-color-orion-10);
+  --mb-color-bg-light: var(--mb-base-color-orion-5);
   --mb-color-bg-white: var(--mb-base-color-white);
-  --mb-color-bg-error: #ed6e6e55;
-  --mb-color-bg-night: #42484e;
-  --mb-color-shadow: rgba(0, 0, 0, 0.13);
-  --mb-color-border: #eeecec;
-  --mb-color-filter: #7172ad;
-  --mb-color-focus: var(--mb-base-color-blue-20);
-  --mb-color-warning: #f9cf48;
+  --mb-color-bg-error: color-mix(
+    in srgb var(--mb-color-error) 33%,
+    transparent
+  );
+  --mb-color-bg-night: var(--mb-base-color-orion-80);
+  --mb-color-shadow: var(--mb-base-color-orion-alpha-20);
+  --mb-color-border: var(--mb-base-color-orion-alpha-20);
+  --mb-color-filter: var(--mb-base-color-octopus-70);
+  --mb-color-focus: var(--mb-base-color-brand-40);
+  --mb-color-warning: var(--mb-base-color-dubloon-30);
   --mb-color-text-primary: var(--mb-color-text-dark);
   --mb-color-text-secondary: var(--mb-color-text-medium);
   --mb-color-text-tertiary: var(--mb-color-text-light);
   --mb-color-text-selected: var(--mb-base-color-white);
   --mb-color-text-hover: var(--mb-base-color-brand-40);
-  --mb-color-text-disabled: var(--mb-base-color-gray-50);
+  --mb-color-text-disabled: var(--mb-base-color-orion-alpha-40);
   --mb-color-text-brand: var(--mb-base-color-brand-40);
   --mb-color-text-white: var(--mb-base-color-white);
   --mb-color-background: var(--mb-base-color-white);
-  --mb-color-background-info: var(--mb-base-color-gray-5);
+  --mb-color-background-info: var(--mb-base-color-orion-5);
   --mb-color-background-selected: var(--mb-base-color-brand-40);
-  --mb-color-background-hover: var(--mb-base-color-brand-10);
-  --mb-color-background-disabled: var(--mb-base-color-gray-10);
+  --mb-color-background-hover: var(--mb-base-color-ocean-10);
```

Comment:
This will break a custom brand color
![Screenshot 2024-11-06 at 11 11 03 PM](https://github.com/user-attachments/assets/d7d78e69-1328-418d-ac55-b11991931124)

I think this should still be brand in most cases.